boards/arc/nsim/nsim_em11d.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: nsim_em11d
+name: EM Nsim simulator
+type: mcu
+simulation: nsim
+arch: arc
+toolchain:
+  - zephyr
```

Comment:
not sure you want to claim zephyr toolchain support, as the relevant features (DSP) require MWDT.
Did you test with zephyr/gnu toolchain? If things work and e.g. only dsplib or mli is compiled with metaware, but zephyr and applications with gnu, that would be great.

Reply:
Actually FXAPI library and MLI_v1.1 (current is v2.0) can be compiled with mwdt into static library, then linked to zephyr application with gnu. MLI_v2.0 can't achieve this. It's good to add this in the future, but currently let's just remove gnu, only use mwdt first.